.python-version

```diff
@@ -0,0 +1 @@
+3.8
```

Comment:
To be removed. I also accidentally added this recently to a repo, comes from `pyenv local`.